Tests/SwiftLintFrameworkTests/ConfigurationTests.swift

```diff
@@ -194,15 +194,74 @@ class ConfigurationTests: XCTestCase {
     func testConfiguresCorrectlyFromDict() {
         let ruleConfiguration = [1, 2]
         let config = [RuleWithLevelsMock.description.identifier: ruleConfiguration]
-        let rules = testRuleList.configuredRules(with: config)
+        // swiftlint:disable:next force_try
+        let rules = try! testRuleList.configuredRules(with: config)
```

Comment:
Ah, my mistake, the `try` has to be in an `XCTAssert*` autoclosure. Lots of our existing tests leverage this.

However, you can mark this test function as `throws`, which will have the same effect.